### ISSUE TITLE
Duplicated requests when using weebhook

### ISSUE THREAD

**Issue by jojomak13** [gotenberg/gotenberg#1085]
I have recently started using the webhooks feature for `forms/chromium/convert/html`.

However, when I send multiple requests sequentially (e.g., **4** requests), each with a **unique** number in the header based on `gotenberg-trace`, I receive the correct number of webhooks (**4** in this case). The issue is that some webhooks are duplicated, sharing the same `gotenberg-trace` and **PDF** content.

![Screenshot 2024-12-22 at 8 54 50 PM](https://github.com/user-attachments/assets/780f933b-59e1-4f85-a072-2476ccd064e4)
![Screenshot 2024-12-22 at 8 55 00 PM](https://github.com/user-attachments/assets/c6238295-9acf-418d-85ce-cd369e8f7a26)

**Comment by gulien** [comment]
Hey @jojomak13 👋

Which version of Gotenberg are you using?

**Comment by jojomak13** [comment]
i use the docker version 8

```yaml
  pdf:
    image: gotenberg/gotenberg:8
    container_name: pdf
    ports:
      - 3000:3000
```

**Comment by gulien** [comment]
Thanks, but do you have the exact version (`/version` route)? 😄

**Comment by jojomak13** [comment]
how can i get it? i am not sure

**Comment by gulien** [comment]
By calling the `/version` route!

**Comment by jojomak13** [comment]
It returns with **404**

![Screenshot 2024-12-22 at 9 57 00 PM](https://github.com/user-attachments/assets/a53c7fb9-3005-447d-9dff-e11deaf54a91)

**Comment by jojomak13** [comment]
I tried also version **7** and it have the same issue

**Comment by starwalkn** [comment]
> I tried also version **7** and it have the same issue

Version route added in Gotenberg 8.5.0. Check that your version is equal to or higher than this one: [8.5.0](https://github.com/gotenberg/gotenberg/releases/tag/v8.5.0)

**Comment by jojomak13** [comment]
I got the version from logs, **V8.0.1**

**Comment by starwalkn** [comment]
> I got the version from logs, **V8.0.1**

You should be using version 8.11.1 and above, this problem is fixed there: [8.11.1 release notes](https://github.com/gotenberg/gotenberg/releases/tag/v8.11.1)
